apisix/plugins/batch-requests.lua

```diff
@@ -179,6 +183,8 @@ local function set_common_header(data)
                 end
             end
         end
+
+        req.headers[real_ip_hdr] = core.request.get_remote_client_ip()
```

Comment:
Is this a feature or bugfix?

Reply:
bugfix, see: https://github.com/apache/apisix/pull/5476/files#diff-b80ee9fead226c0432f9e78cf5cae941641f9f685c49002f6a51310dd7134892R187